scimath/StatsFramework/FitToHalfStatistics.h

```diff
@@ -256,18 +256,18 @@ class FitToHalfStatistics
     StatsData<AccumType> _statsData;
     Bool _doMedAbsDevMed{False}, _rangeIsSet{False};
     // these are the max and min for the real portion of the dataset
-    CountedPtr<AccumType> _realMax{}, _realMin{};
+    std::shared_ptr<AccumType> _realMax{}, _realMin{};
     Bool _isNullSet{False};
     // This is used for convenience and performance. It should always
     // be the same as the _range value used in the base
     // ConstrainedRangeStatistics object
-    CountedPtr<std::pair<AccumType, AccumType> > _range{nullptr};
+    std::shared_ptr<std::pair<AccumType, AccumType>> _range{nullptr};
```

Comment:
Probably just remove the unnecessary nullptr here, since that's the default if not specified.

Reply:
done